src/test/java/com/gfl/client/model/ProxyConfigHolderTest.java

```diff
@@ -0,0 +1,129 @@
+package com.gfl.client.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyConfigHolderTest {
```

Comment:
It does not test cases of `Valid, NotNull` coverage.
you can use for validation:
```
    private final ValidatorFactory validatorFactory = Validation.buildDefaultValidatorFactory();
    private final Validator validator = validatorFactory.getValidator();
 ```